build.sh

```diff
@@ -34,13 +36,35 @@ case $i in
     FINAL=YES
     shift
     ;;
+    *)
+    VERSION=$i
 esac
 done
 
 [ "${FINAL}" == "NO" ] && BOSH_OPTS="--force" || BOSH_OPTS="--final"
+[ "${FINAL}" == "YES" ] && git clean -xdf
 [ "${DEBUG}" == "YES" ] && export BOSH_LOG_LEVEL=debug
 [ "${DEBUG}" == "YES" ] && MVN_OPTS='-B' || MVN_OPTS='-q'
 
+# Checking 'tile' binary version
+currentver=$(tile -v | cut -d" " -f3)
+requiredver=13.1
+ if [ "$(printf '%s\n' "${requiredver}" "${currentver}" | sort -V | head -n1)" != "${requiredver}" ]; then 
```

Comment:
okay for now. But better to check if current ver is greater than required version rather than !=.